yarn.lock

```diff
@@ -5934,22 +5934,14 @@
   dependencies:
     "@types/react" "*"
 
-"@types/react@*", "@types/react@^16.3.0", "@types/react@^16.4.18", "@types/react@^16.7.17", "@types/react@^16.8.0", "@types/react@^16.8.19":
+"@types/react@*", "@types/react@^16.3.0", "@types/react@^16.4.18", "@types/react@^16.7.17", "@types/react@^16.8.0", "@types/react@^16.8.19", "@types/react@^16.9.17":
```

Comment:
Just want to make sure this is expected.

Reply:
It does seem strange that yarn is doing this. I think this means that only 1 version of `@types/react` is installed though, so I think it's what we want.